yesod-auth/Yesod/Auth/Message.hs

```diff
@@ -60,6 +60,7 @@ data AuthMessage =
     | ProvideIdentifier
     | SendPasswordResetEmail
     | PasswordResetPrompt
+    | CurrentPassword
```

Comment:
Using the English string initially makes sense. After this gets merged in, you could send a message to the mailing list asking people to provide translations for their languages.